tests/cli/create/create_tests.py

```diff
@@ -76,9 +80,113 @@ def test_009_create_project_with_appid(self):
         Tns.create(app_name=Settings.AppName.DEFAULT, template=Template.HELLO_WORLD_JS.local_package,
                    app_data=Apps.HELLO_WORLD_JS, update=False, app_id='org.nativescript.MyApp')
 
+    def test_010_create_project_with_named_app(self):
+        """Create app named 'app' should not be possible without --force option"""
+        Tns.create(app_name=Settings.AppName.APP_NAME, template=Template.HELLO_WORLD_JS.local_package,
+                   app_data=Apps.HELLO_WORLD_JS, force=True, update=False)
+        json = App.get_package_json(app_name=Settings.AppName.APP_NAME)
+        assert json['nativescript']['id'] == 'org.nativescript.{0}'.format(Settings.AppName.APP_NAME)
+
+        # Create project should fail without --force
+        result = Tns.create(app_name=Settings.AppName.APP_NAME, verify=False, update=False)
+        assert "You cannot build applications named '" + Settings.AppName.APP_NAME + "' in Xcode." in result.output
+
+    def test_011_create_app_default(self):
+        Folder.clean(os.path.join(Settings.AppName.APP_NAME))
+        Tns.create(app_name=Settings.AppName.DEFAULT, default=True, update=False)
+
+    def test_012_create_app_remove_app_resources(self):
```

Comment:
Isn't it kind of same with `test_016_create_project_with_no_app_resoruces`?
May be we should talk with @rosen-vladimirov to see if we need both of them.